### ISSUE TITLE
resetHandlers() doesn't clean requests array

### ISSUE THREAD

**Issue by loicsimon** [spreaker/node-mock-http-server#24]
We are bullet proofing an API which calls another API. This second API is mocked by using mock-http-server.
We thought that by calling `resetHandlers()` and recreating an handler before each unit test, it would also clear previously received requests but it doesn't.

Is there any `cleanRequests()` function? 
Wouldn't it be great to clean the requests array when we use `resetHandlers()` (see below) ?
```javascript
    this.resetHandlers = function() {
        requests = [];
        handlers = [];
    };
```

BTW, we don't want to stop/start a server before each unit test. It takes a bit of time to stop the server.

Thank you.

**Comment by loicsimon** [comment]
If you're interested, you can find a solution with pull request #25

**Comment by loicsimon** [comment]
This issue is resolved by PR #25 which was merged and published through v1.4.2
